common/mails/courses/index.ts

```diff
@@ -183,3 +185,45 @@ export async function sendParticipantCourseCertificate(participant: Pupil, cours
     //send mail 🎉
     await sendTemplateMail(mail, participant.email);
 }
+
+export async function sendPupilCourseSuggestion(course: Course | Prisma.course, subcourse: Subcourse | Prisma.subcourse) {
+    const minGrade = subcourse.minGrade;
+    const maxGrade = subcourse.maxGrade;
+
+    let lectures = await prisma.lecture.findMany({ where: { subcourseId: subcourse.id }, orderBy: { start: 'asc' }, take: 1 });
+    if (lectures.length == 0) {
+        logger.info('No lectures found: no suggestions sent for subcourse ' + subcourse.id + ' of course ' + course.name);
+        return;
+    }
+
+    let firstLecture = lectures[0].start;
```

Comment:
I think if you put this logic into a function the code would be more readable, because the function name would say what is this loop's intention

Reply:
maybe just create a getCourseStartDate function and put all lecture stuff there